### DISCUSSION TITLE
GUI and Frontend working on same port

### DISCUSSION THREAD

**Discussion by dgilmore82** [Velocidex/velociraptor#3051] [Q&A] [answered]
Hello,
 
 I wanted to know if it is possible to have clients connect to Velociraptor and access the GUI on port 443?

**Comment by sourabhsharma24** [comment, marked as answer]
> I wanted to know if it is possible to have clients connect to Velociraptor and access the GUI on port 443?
Yes, it is possible to have clients connect to Velociraptor and access the GUI on port 443. Port 443 is commonly used for HTTPS, and it provides secure communication over the internet. By configuring Velociraptor to listen on port 443, you can ensure that client connections to the GUI are encrypted and secure, protecting sensitive information and ensuring data integrity during communication.
